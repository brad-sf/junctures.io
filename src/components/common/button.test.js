import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Button from './button';

it('render correctly', () => {
	const tree = renderer
		.create(<Button text="Next"/>)
		.toJSON();

	expect(tree).toMatchSnapshot();
});


