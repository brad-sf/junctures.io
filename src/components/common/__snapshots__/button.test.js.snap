// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
.c0 {
  display: inline-block;
  padding: 0 16px;
  height: 36px;
  min-width: 64px;
  border: none;
  border-radius: 2px;
  line-height: 36px;
  font-size: 14px;
  color: #000;
  background-color: hsla(0,0%,50%,0.1);
  cursor: pointer;
  -webkit-transition: background-color ease-in-out 200ms;
  transition: background-color ease-in-out 200ms;
}

.c0:active {
  background-color: hsla(0,0%,50%,0.2);
}

<button
  className="c0"
  type=""
>
  Next
</button>
`;
